### PR TITLE
Dev 163 create industry page

### PR DESCRIPTION
**Pull request by Max-vS**
Can still be optimized, but just to get a solid version out there.

### CODE REVIEW

File: pages/industry.tsx [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+          text: "During this AI project, our team worked broadly on conversational speech recognition technology for the medical domain. This involved the assembly of a German speech dataset, training (and/or fine-tuning) large modern speech models on our compute infrastructure and evaluating their effectiveness relative to current cloud offerings.",
+        },
+      ],
+      organization: "Cognote",
+      organizationLink: "https://www.cognote.ai/",
+      time: "spring 2022",
+    },
+    {
+      title: "Presize.ai - Clothing size recommender systems",
+      image: "/assets/industry/project_cards/Unbenannt 2.jpg",
```
Comment (simon-hng) [inline pages/industry.tsx]: The image set as source here does not exist

File: pages/industry.tsx [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+        imageSrc="/assets/industry/header/infineon_industry.jpeg"
+        titleImageSrc="/assets/industry/header/industry_header_no_date.svg"
+        subtitle="Your stepping stone to a career in AI"
+      />
+      <div>
+        <div className="container mx-auto -mb-8 flex justify-center md:hidden">
+          {
+            <Popover className="relative p-10">
+              <Popover.Button className="inline-flex items-center gap-x-1 rounded-xl p-4 text-lg font-medium leading-6 text-purple-950 shadow-lg shadow-purple-900/10 ring-1 ring-gray-400/10 transition-all duration-300 hover:shadow-purple-900/20">
+                <span>Options</span>
```
Comment (simon-hng) [inline pages/industry.tsx]: IMO we can just stick with the tabs here

**Review by simon-hng — APPROVED** [review]
Very cool! Thank you max

File: pages/industry.tsx (hunk 2 above)
Comment (Max-vS) [reply pages/industry.tsx]: They rearrange weirdly when screen is shrunk.

File: pages/industry.tsx [hunk 3]
```diff
@@ -0,0 +1,680 @@
+import Image from "next/image";
+import Link from "next/link";
+import PictureHero from "../components/BannerHero";
+import Section from "@ui/Section";
+import { Fragment, useRef, useState } from "react";
+import { Dialog, Transition, Popover } from "@headlessui/react";
+import {
```
Comment (simon-hng) [inline pages/industry.tsx]: - [ ] TODO: refactor to use fontawesome icons

File: pages/industry.tsx [hunk 4]
```diff
@@ -0,0 +1,680 @@
+import Image from "next/image";
+import Link from "next/link";
+import PictureHero from "../components/BannerHero";
+import Section from "@ui/Section";
+import { Fragment, useRef, useState } from "react";
+import { Dialog, Transition, Popover } from "@headlessui/react";
```
Comment (simon-hng) [inline pages/industry.tsx]: - [ ] TODO: Refactor to use radix-ui

**Review by simon-hng — CHANGES_REQUESTED** [review]
I talked with Mohammed and we decided that we want to resolve the todos before merging

File: pages/industry.tsx (hunk 2 above)
Comment (simon-hng) [reply pages/industry.tsx]: - [ ] TODO: Replace with [radix-ui tabs](https://www.radix-ui.com/docs/primitives/components/tabs)

**Review by simon-hng — APPROVED** [review]

**Pull request merged**